webpack.config.babel.js

```diff
@@ -203,6 +204,10 @@ if (isProduction) {
         warnings: false,
       },
     }),
+    new CopyWebpackPlugin([
+      { from: 'packages/bpk-docs/src/README.md', to: 'README.md' },
```

Comment:
Couldn't we just create the `README.md` in https://github.com/backpack/backpack.github.io once and then leave it?

Reply:
Travis will force push to the target branch so it'll be nuked

Reply:
Really? So it will always just have a single commit?

Reply:
according to the docs yes
https://docs.travis-ci.com/user/deployment/pages/

Reply:
Cool